src/status_im/ethereum/json_rpc.cljs

```diff
@@ -72,3 +76,8 @@
            on-success)
          :on-error
          on-error}))
+
+;; effects
+(re-frame/reg-fx
```

Comment:
if you go that path the effect needs to be plural and added to the list of mergeable effects, otherwise it will probably happen

Reply:
done